packages/sources/polygon/src/endpoint/tickers.ts

```diff
@@ -145,7 +145,12 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   const response = await Requester.request<ResponseSchema>(options)
   if (Array.isArray(from) || Array.isArray(to))
     return handleBatchedRequest(jobRunID, request, response, ['min', 'c'])
-  const result = Requester.validateResultNumber(response.data.tickers[0], ['min', 'c'])
+  const result = Requester.validateResultNumber(
+    response.data.tickers[0],
+    ['min', 'c'],
+    undefined,
+    'Result could not be found from response.  This is expected for some pairs during the weekend as the market is closed.',
```

Comment:
Maybe mentioning "Data Provider" directly will implicitly do so, I don't think there's a process in place for data issues like we have for EA ones (yet). Also I'd add that this can happen during some holidays as well